cmd/mdmctl/config.go

```diff
@@ -80,21 +81,35 @@ func setCmd(cfg *ClientConfig, args []string) error {
 	return SaveClientConfig(cfg)
 }
 
-func clientConfigPath() (string, error) {
+func clientConfigDirPath() (string, error) {
 	usr, err := user.Current()
 	if err != nil {
 		return "", err
 	}
-	path := usr.HomeDir + "/.micromdm/default.json"
-	return path, nil
+	return path.Join(usr.HomeDir, ".micromdm"), nil
+}
+
+func clientConfigPath() (string, error) {
+	dir, err := clientConfigDirPath()
+	return path.Join(dir, "default.json"), err
```

Comment:
./shrug The extra call to path.Join with a wrapped func looks more awkward to me, but I'm fine with it if that's your preference.